### PR TITLE
Example of creating a user in Active Directory

### PR DESCRIPTION
**Pull request by No9**
This has been raised in a couple of issues and while people can probably piece together a solution I thought this might be handy to have in one space.

### CODE REVIEW

File: docs/examples.md [hunk 1]
```diff
@@ -535,3 +535,54 @@ To test out this example, try:
...
+
+client.bind('Administrator@test.ldap.com', 'password', function(err) {
+    if(err) {
+        console.log("error from bind ",err);
+        return;
+    }
+    console.log('LDAP Binding Successful');
+
+    // Surround the password with " " and encode it with utf16le
+    const buf = new Buffer('"pwd1234$$"', 'utf16le');
```
Comment (jsumners) [inline docs/examples.md]:
```suggestion
    const buf = Buffer.from('"pwd1234$$"', 'utf16le');
```

https://nodejs.org/dist/latest-v10.x/docs/api/buffer.html#buffer_class_method_buffer_from_string_encoding

**Comment by tastypackets** [comment]
I'd be willing to help create some ActiveDirectory documentation as well, since it has its quirks compared to LDAP.

Would it make sense to maybe add a specific ActiveDirectory examples / docs?

**Comment by jsumners** [comment]
@tastypackets yes, that would be fine. Thank you.

**Comment by jsumners** [comment]
⚠️ This issue has been locked due to age. If you have encountered a recent
problem that seems to be covered by this issue, please open a new issue.

Please include a [minimal reproducible example](https://en.wikipedia.org/wiki/Minimal_reproducible_example)
when opening a new issue.

**Pull request closed without merging**